.vscode/launch.template.json

```diff
@@ -10,7 +22,7 @@
       "type": "node",
       "protocol": "inspector",
       "request": "launch",
-      "name": "Mocha Tests (currently opened test)",
+      "name": "Mocha Tests (currently opened test)",
```

Comment:
These invisible chars weren't spaces. `/shrug`?

Reply:
I think I copied this from Teams from Ron 😂